contents/blog/product-360.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2022-01-18
+title: I used to think product people aren't needed. I was wrong
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog", "Engineering"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+---
+
+When PostHog was fresh out of YC last year, there were two things we focused on: (i) build and (ii) talk to customers. Hiring a product person felt like it would slow us down. Engineers on our team would speak to our users (also engineers) - surely they'd know what's best to prioritize? We didn't want our engineers to be "shielded" from our customers.
+
+By [PostHog's series B](/blog/15-million-series-b), our team showed me that I was totally wrong. Here's why that was the case, what we did about it, and the results.
+
+>James Hawkins is PostHog Co-Founder & CEO. Connect with him on [LinkedIn](https://www.linkedin.com/in/j-hawkins/) and follow him on [Twitter](https://www.linkedin.com/in/j-hawkins/).
+
+## Why I used to think this way
+
+My previous experience of product management:
+
+* Here's a list of features the enterprise sales team wants
+* How much pipeline does each of these features influence?
+* _Builds as much of the above as possible whilst trying to manage existing customer requests_
+
+Led to:
+
+* Lots of initial enterprise contracts
+* A product users find frustrating to use
+* No word of mouth growth
+* Customer churn in the long term
+
+## It starts with expectation setting
```

Comment:
What is this section trying to say? I'm confused. There's a few gifs, and a mention of Goodharting, so I guess you're trying to say something like.... what you mention in the above section has even more negative consequences?